src/gui/editorwidgets/qgsvaluerelationwidgetwrapper.cpp

```diff
@@ -434,3 +435,25 @@ void QgsValueRelationWidgetWrapper::setEnabled( bool enabled )
   else
     QgsEditorWidgetWrapper::setEnabled( enabled );
 }
+
+void QgsValueRelationWidgetWrapper::setConfig( const QVariantMap &config )
```

Comment:
Is this code duplicated?
Can this be encapsulated in a static method in a static method in here to resolve the layer?

Reply:
Yeah, I thought about that but I gave up after struggling for a while about where to put it, where would you put it, in the cfgdlg?

Reply:
First guess would be the wrapper (and not the dlg). Or evenmore look into https://github.com/qgis/QGIS/pull/30151#discussion_r291966938 first .

Reply:
looking again at it, we might have a const-ness difference between the two usages. The call to 
`mLayerName->setLayer( lyr );` takes a not-const.